addons/mail/static/src/components/attachment/attachment.scss

```diff
@@ -60,6 +64,8 @@
 }
 
 .o_Attachment_image {
+    display: flex;
```

Comment:
If I'm not mistaken `display: flex` is for the container, image is just a flex children and doesn't need it.

Reply:
It was a way to avoid that :
![image](https://user-images.githubusercontent.com/48204996/90005456-8b180100-dc97-11ea-963d-50bcfb54def2.png)

by saying "ok you're a flex container and you cannot shrink"

It could also be done with a min-width:34px; but need to use a variable for 34 px I suppose.
What do you prefer ?

Reply:
From by testing just `flex-shrink: 0;` is enough to avoid the issue on the screenshot, I don't see a difference when I remove `display: flex;`.